src/book/03-guides/02-tools/03-webots.mdx

```diff
@@ -76,51 +76,74 @@ sudo apt-get install cmake-curses-gui libprotobuf-dev protobuf-compiler libeigen
 
 ## Install Webots
 
-1. Open a terminal and swap to your home directory (note that Webots should be cloned in your home directory). Then clone the repository and change into the cloned directory:
+1. Open a terminal, clone the NUWebots repository and change into the cloned directory:
 
    ```bash
    cd ~
-   git clone --branch nubots-changes --single-branch --recurse-submodules -j$(nproc) https://github.com/NUbots/webots.git
-   cd webots
+   git clone --recurse-submodules -j$(nproc) https://github.com/NUbots/NUWebots.git
+   cd NUWebots
+   ```
+
+   The NUWebots repo contains both NUbots-specific Webots files and the files needed to install Webots itself.
+   The first command can be changed to any directory you would like. If you're not sure, just follow what is in this guide.
+
+2. This step assumes you have done the previous step and are now in the NUWebots directory. Add the `WEBOTS_HOME` env variable to your `.bashrc` file:
+
+   ```
+   echo export WEBOTS_HOME=$(pwd)/webots >> ~/.bashrc
+   source ~/.bashrc
    ```
 
-   By using these commands, you'll be on the correct branch (`nubots-changes`) and you'll have the required submodules.
+3. Update the submodules for the Webots subtree
 
-2. Install Webots build dependencies:
+   ```bash
+   git submodule update --init
+   ```
+
+4. Move into the webots directory and install the Webots build dependencies:
 
    ```bash
+   cd webots
    sudo ./scripts/install/linux_compilation_dependencies.sh
    ```
 
-3. Build Webots:
+5. Build Webots:
 
    ```bash
    make -j$(nproc)
    ```
 
-4. Add the `WEBOTS_HOME` env variable to your `.bashrc` file:
+## Running the Official RoboCup Simulation in Webots
 
+### Set Up the Protos
+
+1. Set up the symlink for the NUgus proto:
+
+   ```bash
+   ln -s $WEBOTS_HOME/../protos/robot/NUgus $WEBOTS_HOME/projects/samples/contests/robocup/protos
    ```
-   echo WEBOTS_HOME=$HOME/webots >> ~/.bashrc
-   ```
 
-5. Install dependencies for the referee:
+   This allows us to use our NUgus robot with the official RoboCup simulation.
+
+### Set Up the Controllers
+
+1. Install dependencies for the referee:
 
    ```bash
-   cd ~/webots/projects/samples/contests/robocup/controllers/referee
+   cd $WEBOTS_HOME/projects/samples/contests/robocup/controllers/referee
    pip3 install -r requirements.txt
    ```
 
-6. Build the Robocup controllers:
+2. Build the Robocup controllers:
 
    ```bash
-   cd ~/webots/projects/samples/contests/robocup
-   WEBOTS_HOME=$HOME/webots make -j$(nproc)
+   cd $WEBOTS_HOME/projects/samples/contests/robocup
+   make -j$(nproc)
```

Comment:
This make requires `$WEBOTS_HOME` to be set, and fails in a very unhelpful way if it's not. could you please make it clear at this step that it does need to be set, for future me and other `.bashrc` zealots.

For context, here is what it says if it's not set:
```bash
$ make
#
# *** controllers ***
# make release player
Makefile:51: /resources/Makefile.include: No such file or directory
make[2]: *** No rule to make target '/resources/Makefile.include'.  Stop.
make[1]: *** [Makefile:21: player.Makefile] Error 2
make: *** [Makefile:19: release] Error 2
```
The way I worked out why it wasn't working was seeing that this line got changed on this diff

Reply:
I've added something about this at the beginning of the `Running the Official RoboCup Simulation in Webots` section